x-pack/plugins/fleet/server/services/epm/packages/install_steps.ts

```diff
@@ -0,0 +1,451 @@
+/*
```

Comment:
nit: that module seems already impressive to me what do you think of creating a submodule with maybe one file per step or some logicial grouping per file? and maybe one day we can have unit tests for every step